README.md

```diff
@@ -51,6 +51,18 @@ Commands:
 
 ---
 
+### Caching
+
+You can use caching by including this module
+
+    include Gris::CacheKey
+
+in your ActiveRecord models, and then cache inside GET requests like this
+
+    offer_event =  Gris.cache.fetch(OfferEvent.cache_key_for(id)) do
+      OfferEvent.find(id)
+    end
```

Comment:
I wonder whether we wouldn't want to additionally add a `.fetch` method to the Gris::CacheKey module (and name it slightly more generally given an expanded scope) to expand its scope with a wrapper for this block.

```offer_event = OfferEvent.fetch params[:id]```

rather than 

```
offer_event =  Gris.cache.fetch(OfferEvent.cache_key_for(id)) do
  OfferEvent.find(id)
end
```